src/module.ts

```diff
@@ -184,158 +184,161 @@ export default defineNuxtModule<ModuleOptions>().with({
       options.client = 'always'
     }
 
-    await nuxt.callHook('api-party:extend', options)
+    // ensure other modules are loaded before we call the hook.
+    // Use imports:sources because modules:done is too late to register auto-imports
+    nuxt.hook('imports:sources', async (sources) => {
```

Comment:
The only hook before this is `pages:routerOptions`, but it's also called twice for whatever reason.